docs/load-balancer/sticky.md

```diff
@@ -0,0 +1,29 @@
+# Sticky Load Balancing
+
+The sticky load balancer selects a single service instance and keeps using it until it fails.
+Then, it selects the next one.
+
+It is possible to configure a backoff time to specify for how long a failing service instance should not be retried.
+
+Precisely, the load balancer works as follows:
+
+* if the previously selected service instance has not failed, and is still available, return it;
+* else if exists, return the first available service instance that has no recorded failure;
```

Comment:
I would reword the if exists: if the instance is still available, but has failed a previous invocation...